src/oxli/kmer_hash.cc

```diff
@@ -49,6 +49,11 @@ Contact: khmer-project@idyll.org
 
 using namespace std;
 
+#define tbl \
+  "                                                                "\
+  /*ABCDEFGHIJKLMNOPQRSTUVWXYZ      abcdefghijklmnopqrstuvwxyz    */\
+  " TVGH FCD  M KN   YSAABW R       TVGH FCD  M KN   YSAABW R"
```

Comment:
this determines how a input base gets translated into its complement. `A->T`, `B->V`, etc. This is a copy&paste from the benchmarks-game code. I already modified it so that `F->F`. Can someone with some science knowledge check if these make sense?

Reply:
I'm curious - beyond A <-> T and C <-> G and N <-> N what is the rest for?

Reply:
TBH I have no idea, this comes straight from the benchmarkgames. My guess is they saw [this](https://en.wikipedia.org/wiki/FASTA_format#Sequence_representation) and added them all I added the `F->F` because of https://github.com/dib-lab/khmer/blob/9323ca4e17bccac9ec1a7d5e8321eeec61cd17cd/tests/test_functions.py#L116

Reply:
@ctb these are all valid base conversions, based on the IUPAC nucleotide code: https://www.bioinformatics.org/sms/iupac.html

Reply:
right, suspected as much, not sure we need them but they will do no harm either...